docs/ad/api.md

```diff
@@ -878,6 +1075,55 @@ PUT _opendistro/_anomaly_detection/detectors/<detectorId>
 }
 ```
 
+To update a historical detector, you need to first stop the detector.
```

Comment:
How about move this sentence up to the description paragraph under `Update detector` title